### PR TITLE
kube-style org automation

### LINKED ISSUE #24: automate org/team membership
https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/ looks like it might be a good way to handle this, open to suggestions though

### PR DESCRIPTION
**Pull request by ilrudie**
Working on getting [peribolos](https://docs.prow.k8s.io/docs/components/cli-tools/peribolos/) running to manage org/team membership in a similar style to Kubernetes and Istio (but ideally without any other Prow components).

Fixes #24

### CODE REVIEW

File: org.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    teams:
+      community-maintainers:
+        description: ""
+        members:
+        - craigbox
+        - danehans
+        - ilrudie
+        - jenshu
+        - linsun
+        privacy: closed
```
Comment (linsun) [inline org.yaml:34]: what does `privacy: closed` mean?

**Review by linsun — COMMENTED** [review]
lgtm

File: org.yaml (hunk 1 above)
Comment (ilrudie) [reply org.yaml:34]: It's a standard bit of the [org config](https://github.com/kubernetes/org/blob/fced6f2e4cb797ef9b466ac04689c2450528c4b8/config/kubernetes/org.yaml#L1275) for this tool.

It comes from the gh json for a team:

```
privacy string
The level of privacy this team should have. The options are:

secret - only visible to organization owners and members of this team.
closed - visible to all members of this organization.
Default: secret
Can be one of: secret, closed
```

It translates to "visible" in the gh webui

**Comment by ilrudie** [comment]
Presently, This would perform a dry run which would not actually modify the org. I did not create the secret required yet either. Permissions needed are not available via the normal GITHUB_TOKEN variable so [gateway-bot](https://github.com/gateway-bot) will need to be made and org admin and then we can generate the required to token to try this out.

File: org.yaml (hunk 1 above)
Comment (linsun) [reply org.yaml:34]: Ah thanks!

File: org.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - jbohanon
+    - jenshu
+    - josh-pritchard
+    - kevin-shelaga
+    - lgadban
+    - Nadine2016
+    - shashankram
+    - sheidkamp
+    - Sodman
+    - stevenctl
```
Comment (linsun) [inline org.yaml:24]: Does this miss a few other non Solo maintainers like kdorosh, Sai, Tyler etc?

Comment (linsun) [reply org.yaml:24]: That sounds good, we can start with it and iterate.

**Comment by ilrudie** [comment]
> Is this ready to be merged?

I think it could merge. Right now it should do nothing. It's not setup to actually modify the org and I did not create the secret credentials so 2 bits away from it being "live". Only thing I can think of is I created a new team which is not reflected in the config. Let me update that right quick and then it could go

**Review by nfuden — APPROVED** [review]

**Pull request merged**